src/main/java/net/minecraftforge/fluids/FluidType.java

```diff
@@ -1151,5 +1170,24 @@ public Properties rarity(Rarity rarity)
             this.rarity = rarity;
             return this;
         }
+
+        /**
+         * Allows this fluid to drip from Pointed Dripstones and fill cauldrons below.
+         *
+         * @param chance the chance that the cauldron below will be filled every time the Pointed Dripstone is randomly ticked
+         * @param dripParticle the particle that spawns randomly from the tip of the Pointed Dripstone when this fluid is above it
+         * @param cauldron the block the Pointed Dripstone should replace an empty cauldron with when it successfully tries to fill the cauldron
+         * @param fillSound the sound that plays when the Pointed Dripstone successfully tries to fill an empty cauldron. If null, no sound will play. Note that if your block class does not extend {@link CauldronBlock}, this sound will not play regardless.
```

Comment:
You should be using a sound action for this. Probably something like `CAULDRON_DRIP`. It can be abstracted away in the body of the method.

Reply:
Will switch to a SoundAction. 
I initially wanted to keep the SoundEvent in the record to make it less confusing to add one, but I suppose it makes more sense if it's a SoundAction since that's how all other fluid sounds are handled